include/seqan3/core/detail/to_string.hpp

```diff
@@ -0,0 +1,34 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Svenja Mehringer <svenja.mehringer AT fu-berlin.de>
+ * \brief Auxiliary for pretty printing of exception messages.
+ */
+
+#pragma once
+
+#include <seqan3/io/stream/debug_stream.hpp>
+
+namespace seqan3::detail
```

Comment:
I thought we don't do nested namespaces?

Reply:
o.0?

Reply:
I did it once and was asked to do
```
namespace seqan3
{

namespace detail
{

bla

} // namespace seqan3::detail

} // namespace seqan3
```

But apparently we do it in many places anyways.
I don't mind either way, so you can just resolve this.

Reply:
No, it's exactly the other way around as can been seen almost everywhere in the library and in the wiki :point_up: